package.json

```diff
@@ -19,13 +19,14 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "postinstall": "cd lib || builder run npm:postinstall",
```

Comment:
Crap. I missed that. Yes, we still need that (haha -- it's the whole reason for the publishr switch!)